src/main/scala/nl.knaw.dans.easy.deposit/docs/AgreementsXml.scala

```diff
@@ -17,15 +17,16 @@ package nl.knaw.dans.easy.deposit.docs
 
 import org.joda.time.DateTime
 
-import scala.util.Try
+import scala.util.{ Failure, Try }
 import scala.xml.Elem
 
 object AgreementsXml extends SchemedXml {
   override val schemaNameSpace = "http://easy.dans.knaw.nl/schemas/bag/metadata/agreements/"
-  override val schemaLocation = "https://easy.dans.knaw.nl/schemas/bag/metadata/agreements/2019/01/agreements.xsd"
+  override val schemaLocation = "https://easy.dans.knaw.nl/schemas/bag/metadata/agreements/2019/09/agreements.xsd"
 
-  def apply(userId: String, dateSubmitted: DateTime, dm: DatasetMetadata, fullname: String): Try[Elem] = {
-    for {
+  def apply(dateSubmitted: DateTime, dm: DatasetMetadata, userProperties: UserInfo): Try[Elem] = {
+    if (userProperties == null) Failure(new IllegalArgumentException("no user attributes"))
```

Comment:
`userProperties` CANNOT be `null`! The type is `UserInfo`, not `Option[UserInfo]`.

Reply:
https://github.com/DANS-KNAW/easy-deposit-api/blob/d614ef71f78e1d04872892a1a2f795136fea81eb/src/test/scala/nl.knaw.dans.easy.deposit/docs/AgreementsSpec.scala#L86-L88